test/e2e/framework/util.go

```diff
@@ -2230,6 +2230,20 @@ func ListSchedulableNodesOrDie(c *client.Client) *api.NodeList {
 	return nodes
 }
 
+func ListReadyNodes(f *Framework) (nodes *api.NodeList, err error) {
```

Comment:
this seems like a common enough thing that we would already have a function that does it, if you've already checked then don't mind this comment